catalog/app/containers/HomePage/index.js

```diff
@@ -27,11 +28,18 @@ const Styler = styled.div`
 
 export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
+    const { team = {} } = config;
+    const header = team.name && typeof team.name === 'string' ?
```

Comment:
Probably safe to assume that if it's set, it's going to be a string.

Reply:
yeah i am preventing `toUpperCase` from blowing up but it's probably overkill